src/App.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":[],"names":[],"mappings":"","file":"App.css"}
```

Comment:
This file should be in gitignore right?

Reply:
Is in the gitignore - will delete any files directly from the branch.